velox/exec/RowContainer.cpp

```diff
@@ -296,6 +303,49 @@ void RowContainer::eraseRows(folly::Range<char**> rows) {
   numFreeRows_ += rows.size();
 }
 
+int32_t RowContainer::pickMemberRows(folly::Range<char**> rows, char** result) {
+  raw_vector<folly::Range<char*>> ranges;
+  ranges.resize(rows_.numRanges());
+  for (auto i = 0; i < rows_.numRanges(); ++i) {
+    ranges[i] = rows_.rangeAt(i);
+  }
+  std::sort(
```

Comment:
Should we sort `starts` instead of the input `rows`?  This makes it safer to use (not changing the input `rows`).  Otherwise we need to mention in the comment that this function (and thus `eraseRows`) can reorder rows.